src/test/java/teammates/test/pageobjects/InstructorCommentsPage.java

```diff
@@ -208,6 +208,52 @@ public void clickResponseCommentDelete(int sessionIdx, int questionIdx, int resp
         waitForPageToLoad();
     }
     
+    /**
+     * Clicks all the headings of the comment panel to either expand / collapse the panel body.
+     */
+    public void clickAllCommentsPanelHeading() {
+        for (WebElement e : browser.driver.findElements(By.cssSelector("div[id^='panel_display-']"))) {
+            e.findElement(By.cssSelector(".panel-heading")).click();
+        }
+    }
+    
+    /**
+     * Checks if the body of all the comment panels are collapsed or expanded.
+     * @param isVisible true to check for expanded, false to check for collapsed.
+     * @return true if all comment panel body are equals to the visibility being checked.
+     */
+    public boolean isAllCommentPanelBodyVisibilityEquals(boolean isVisible) {
```

Comment:
You can make this method private and give two public methods `areCommentsVisible` and `areCommentsHidden`